problem_builder/public/js/student_answers_dashboard.js

```diff
@@ -0,0 +1,299 @@
+
+    var ResultsView = Backbone.View.extend({
+
+        render: function() {
+            this._insertRecords(this.collection.getCurrentPage(true));
+            this._updateControls();
+            this.$('#total-pages').text(this.collection.getTotalPages() || 0);
+            return this;
+        },
+
+        _insertRecords: function(records) {
+            var tbody = this.$('tbody');
+            tbody.empty();
+            records.each(function(result, index) {
+                var row = $('<tr>');
+                if (index % 2 === 0) {
+                    row.addClass('even');
+                }
+                _.each(Result.properties, function(name) {
+                    row.append($('<td>').text(result.get(name)));
+                });
+                tbody.append(row);
+            }, this);
+            if (this.collection.getTotalPages()) {
+                this.$('#current-page').text(this.collection.getCurrentPage());
+            } else {
+                this.$('#current-page').text(0);
+            }
+        },
+
+        events: {
+            'click #first-page': '_firstPage',
+            'click #prev-page': '_prevPage',
+            'click #next-page': '_nextPage',
+            'click #last-page': '_lastPage'
+        },
+
+        _firstPage: function() {
+            this._insertRecords(this.collection.getFirstPage());
+            this._updateControls();
+        },
+
+        _prevPage: function() {
+            if (this.collection.hasPreviousPage()) {
+                this._insertRecords(this.collection.getPreviousPage());
+            }
+            this._updateControls();
+        },
+
+        _nextPage: function() {
+            if (this.collection.hasNextPage()) {
+                this._insertRecords(this.collection.getNextPage());
+            }
+            this._updateControls();
+        },
+
+        _lastPage: function() {
+            this._insertRecords(this.collection.getLastPage());
+            this._updateControls();
+        },
+
+        _updateControls: function() {
+            var currentPage = this.collection.getCurrentPage(),
+                totalPages = this.collection.getTotalPages(),
+                firstPage = '#first-page',
+                prevPage = '#prev-page',
+                nextPage = '#next-page',
+                lastPage = '#last-page',
+                all = [firstPage, prevPage, nextPage, lastPage],
+                backward = [firstPage, prevPage],
+                forward = [nextPage, lastPage];
+            if (!totalPages || totalPages === 1) {
+                this._disable(all);
+            } else {
+                if (currentPage === 1) {
+                    this._disable(backward);
+                    this._enable(forward);
+                } else if (currentPage === totalPages) {
+                    this._enable(backward);
+                    this._disable(forward);
+                } else {
+                    this._enable(all);
+                }
```

Comment:
Nit: If you get rid of `_disable` and make `_enable` accept a boolean parameter, you could simplify this:

```
this._enable(backward, currentPage > 1);
this._enable(forward, currentPage < totalPages);
```